modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -73,15 +73,29 @@ public String getIcon() {
 
 	public String getTitle() {
 		String title = _title;
-
 		HorizontalCard horizontalCard = getHorizontalCard();
 
 		if ((_title == null) && (horizontalCard != null)) {
 			title = horizontalCard.getTitle();
 		}
 
-		return LanguageUtil.get(
-			TagResourceBundleUtil.getResourceBundle(pageContext), title);
+		if (isLocalized()) {
+			title = LanguageUtil.get(
+				TagResourceBundleUtil.getResourceBundle(pageContext), title);
+		}
+
+		return title;
+	}
+
+	public boolean isLocalized() {
+		boolean localized = true;
+		HorizontalCard horizontalCard = getHorizontalCard();
+
+		if (horizontalCard != null) {
+			localized = horizontalCard.isLocalized();
+		}
+
+		return localized;
```

Comment:
This is technically correct, but I'd still prefer if we added a new tag attribute. In general, we are keeping the same set of attributes and model properties. A model object, `HorizontalCard` in this case, is just for convenience when coding, and arguably better mental model. When devs use it, they don't have to set individual tag attributes in JSP, but just one `HorizontalCard` Java object, passed in through the `horizontalCard` tag attribute. But, if somebody prefers setting direct tag attributes, they should have the same capability as when setting the model object. They should be able to set
```jsp
<clay:horizontal-card
    localized="<%= true %>"
/>
```

Reply:
Thank you for explaining, I corrected source as you guided. Do you want me to add the localized attribute to liferay-clay.tld file for both LinkTag and HorizontalCardTag?

Reply:
Yep!

Reply:
I added it.
Thank you!